test/e2e/es_cluster_logging.go

```diff
@@ -196,7 +199,7 @@ func ClusterLevelLoggingWithElasticsearch(c *client.Client) {
 					{
 						Name:    "synth-logger",
```

Comment:
But you are right. I could use `name` also for the container name. It helped me to avoid confusion to keep the container name unrelated to the pod name. We use the container name to help wait on pod success.